dataload/extras/json2sssom/src/main/java/CurieMap.java

```diff
@@ -25,9 +25,10 @@ public CurieMap() {
         addMapping("owl", "http://www.w3.org/2002/07/owl#");
         addMapping("skos", "http://www.w3.org/2004/02/skos/core#");
         addMapping("oboInOwl", "http://www.geneontology.org/formats/oboInOwl#");
-        /* #   rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
-#   rdfs: http://www.w3.org/2000/01/rdf-schema#
-#   sssom: https://w3id.org/sssom/*/
+        addMapping("chebi", "http://purl.obolibrary.org/obo/chebi/");
```

Comment:
For what is this CURIE used? Its an unfamiliar syntax, as  so far, I only know of:

CHEBI: http://purl.obolibrary.org/obo/CHEBI_

or 

chebi: http://purl.obolibrary.org/obo/chebi#

Reply:
That’s a mistake! Thank you!